src/threads-api.ts

```diff
@@ -30,19 +30,6 @@ export class ThreadsAPI {
     origin: 'https://www.threads.net',
     pragma: 'no-cache',
     referer: `https://www.threads.net/@${username}`,
-    'sec-ch-prefers-color-scheme': 'dark',
-    'sec-ch-ua': '"Not.A/Brand";v="8", "Chromium";v="114", "Google Chrome";v="114"',
-    'sec-ch-ua-full-version-list':
-      '"Not.A/Brand";v="8.0.0.0", "Chromium";v="114.0.5735.198", "Google Chrome";v="114.0.5735.198"',
-    'sec-ch-ua-mobile': '?0',
-    'sec-ch-ua-platform': '"macOS"',
-    'sec-ch-ua-platform-version': '"13.0.0"',
-    'sec-fetch-dest': 'empty',
-    'sec-fetch-mode': 'cors',
-    'sec-fetch-site': 'same-origin',
-    'user-agent':
-      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36',
-    'viewport-width': '1728',
     'x-asbd-id': '129477',
     'x-fb-lsd': 'NjppQDEgONsU_1LCzrmp6q',
     'x-ig-app-id': '238260118697367',
```

Comment:
Can't we remove this(`x-asbd-id`, `x-fb-lsd`, and `x-ig-app-id`) and add those headers back?
context: https://github.com/junhoyeo/threads-api/pull/4#discussion_r1254573262

Reply:
If we were to delete that key now, the library would not work at all.

Reply:
gotcha, I'll just merge this PR for now then.